packages/web-client/app/components/card-pay/deposit-workflow/confirmation/index.hbs

```diff
@@ -25,7 +25,7 @@
         <CardPay::AccountDisplay
           class="card-pay-confirmation__transaction-wallet"
           @size="small"
-          @name={{concat this.layer2Network.chainName " wallet"}}
+          @name={{concat this.layer1Network.chainName " wallet"}}
```

Comment:
😯

Reply:
Yep, this is the top half for:
![image](https://user-images.githubusercontent.com/39579264/124004237-67be8780-da0a-11eb-9ae3-88b44ec891a2.png)